tool/teleport/common/teleport.go

```diff
@@ -537,9 +537,9 @@ func onConfigDump(flags dumpFlags) error {
 
 	if configPath != "" {
 		if modules.GetModules().BuildType() == modules.BuildOSS {
-			fmt.Printf("Wrote config to file %q. Now you can start the server. Happy Teleporting!\n", configPath)
+			fmt.Printf("Wrote config to file %q. Now you can start the server. Happy Idemeuming!\n", configPath)
 		} else {
-			fmt.Printf("Wrote config to file %q. Add your license file to %v and start the server. Happy Teleporting!\n", configPath, flags.LicensePath)
+			fmt.Printf("Wrote config to file %q. Add your license file to %v and start the server. Happy Idemeuming!\n", configPath, flags.LicensePath)
```

Comment:
Remove  Happy Idemeuming
Teleporting is actual verb, idemeuming is not.

Reply:
+1.